cms/static/js/views/active_video_upload_list.js

```diff
@@ -23,10 +23,18 @@ define([
                 'drop .file-drop-area': 'dragleave'
             },
 
+            uploadHeader: gettext('Upload Videos'),
+            uploadText: HtmlUtils.interpolateHtml(
+                gettext('Drag and drop videos here, or click this space to {spanStart}browse your computer{spanEnd}.'),
```

Comment:
My suggestion is `Drag and drop videos here, or click here to browse your computer`